types/node/v14/globals.d.ts

```diff
@@ -111,11 +111,21 @@ declare var AbortController: {
     new(): AbortController;
 };
 
-declare var AbortSignal: {
+interface AbortSignalConstructor {
     prototype: AbortSignal;
     new(): AbortSignal;
+    /**
+     * Returns an AbortSignal that will automatically abort after a specified time.
+     *
+     * @param time The "active" time in milliseconds before the returned AbortSignal will abort.
+     *
+     * @since v17.3.0, v16.14.0
```

Comment:
Why add it to v14 when this states it's not available in 14.x?

Reply:
I'm actually running into issues getting this change to pass the type checker. Any tips?